### PR TITLE
Add readmes for System.IO.Compression*

### PR DESCRIPTION
**Pull request by jozkee**

### CODE REVIEW

**Review by carlossanlop — APPROVED** [review]
Descriptions look good, links look good, see alsos look good. :shipit:

File: src/libraries/System.IO.Compression.ZipFile/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Documentation can be found at https://learn.microsoft.com/dotnet/api/system.io.compression.zipfile and https://learn.microsoft.com/dotnet/api/system.io.compression.zipfileextensions.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../../libraries/README.md#secondary-bars)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-System.IO.Compression+label%3A%22help+wanted%22) issues. We don't have a specific label for ZipFile issues so you will need to read through the list and see if something is available.
+
+## Deployment
+`System.IO.Compression.ZipFile` is included in the shared framework and also provided as a [NuGet package](https://www.nuget.org/packages/System.IO.Compression.ZipFile).
```
Comment (teo-tsirpanis) [inline src/libraries/System.IO.Compression.ZipFile/README.md:13]: This package is for .NET Standard 1.x and should not be used in anything newer. See [another example](https://github.com/dotnet/runtime/tree/main/src/libraries/System.Reflection.Primitives#deployment). You should also reword the `System.Diagnostics.Process` README you just added.

Comment (ViktorHofer) [reply src/libraries/System.IO.Compression.ZipFile/README.md:13]: Right, we don't generate a package for `System.IO.Compression` anymore. Either please remove this part of the sentence or make it clear that the package is from the previous era of .NET

Comment (Jozkee) [reply src/libraries/System.IO.Compression.ZipFile/README.md]: Let me know if the new text in https://github.com/dotnet/runtime/pull/78645/commits/cea638614bda534b24079b95f230a707ceb048d1 is good enough or if it needs further clarification.

Comment (teo-tsirpanis) [reply src/libraries/System.IO.Compression.ZipFile/README.md]: Looks fine, thanks.

**Review by ViktorHofer — APPROVED** [review]

**Pull request merged**